shell/tsconfig.json

```diff
@@ -1,14 +1,11 @@
 {
   "compilerOptions": {
-    "composite": true,
+    "composite": false,
```

Comment:
why set to false

Reply:
![image](https://user-images.githubusercontent.com/5175455/121299162-907dc080-c927-11eb-9087-7bc95ff937ce.png)

Reply:
![image](https://user-images.githubusercontent.com/5175455/121299281-c1f68c00-c927-11eb-8184-045987005bca.png)

Reply:
in tsconfig.json of shell, we just need to include source file of shell.  In monorepo, shell refer code like `core/main` which is under tsconfig root, and not `include` either. So set composite to false then ts will find related code automatically